backend/package.json

```diff
@@ -109,6 +109,23 @@
 					"description": "The verbosity of logging according to the following order: trace > debug > info > warn > error > fatal > off.",
 					"scope": "resource"
 				},
+				"ApplicationWizard.TargetFolder": {
+					"type": "string",
+					"default": "/home/user/projects",
+					"description": "Set the default target folder path for the project.",
+					"scope": "resource"
+				},
+				"ApplicationWizard.Workspace": {
+					"type": "string",
+					"enum": [
+						"Open the project in a new workspace",
+						"Open the project in a multi-root workspace",
+						"Create the project and close it for later use"
+					],
+					"default": "Open the project in a new workspace",
+					"description": "Define the preferred behavior upon project creation.",
+					"scope": "resource"
+				},
 				"ApplicationWizard.sourceLocationTracking": {
 					"type": "boolean",
 					"default": false,
```

Comment:
ApplicationWizard -> TemplateWizard

Reply:
in the Preferences the string should stay "Applicaton Wizard"